tuda_workspace_scripts/discovery.py

```diff
@@ -179,6 +179,36 @@ def _create_cyclonedds_config_xml(peers: list[str]) -> str:
     return template.render(peers=peers)
 
 
+def print_discovery_config():
+    print(f"RMW Implementation: {RMW}")
+    if RMW == "rmw_zenoh_cpp":
+        print_zenoh_discovery_config()
+    elif RMW == "rmw_cyclonedds_cpp":
+        print_cyclonedds_discovery_config()
+    elif RMW:
+        raise NotImplementedError(f"Discovery is not implemented for RMW {RMW}")
+    else:
+        raise RuntimeError("RMW_IMPLEMENTATION is not set.")
+
+
+def print_cyclonedds_discovery_config():
+    if os.path.exists(CYCLONEDDS_URI):        
+        print(f"Configuration file: {CYCLONEDDS_URI}")
+        with open(CYCLONEDDS_URI, "r") as file:
+            print_info(file.read())
+    else:
+        print_warn(f"Configuration file not found: {CYCLONEDDS_URI}")
+
+
+def print_zenoh_discovery_config():
+    if os.path.exists(ZENOH_ROUTER_CONFIG_PATH):
+        print(f"Configuration file: {ZENOH_ROUTER_CONFIG_PATH}")
+        with open(ZENOH_ROUTER_CONFIG_PATH, "r") as file:
+            print_info(file.read())
```

Comment:
I think it would be better to print the rmw and config file as info and the config file with regular print.
The dark blue is not a great contrast in my terminal for long outputs.